packages/pytest-simcore/src/pytest_simcore/postgres_service.py

```diff
@@ -70,6 +71,13 @@ def drop_template_db(postgres_engine: sa.engine.Engine) -> None:
     execute_queries(postgres_engine, queries)
 
 
+@pytest.yield_fixture(scope="module")

```

Comment:
just see that now. what does yield_fixture mean exactly?
when I look for it the first result tells me it's deprecated (see [here](https://docs.pytest.org/en/stable/yieldfixture.html))

Reply:
My bad, did not bother to check an internet suggestion.